Tables/Replace-cell-content/.NET/Replace-cell-content/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+
+using (FileStream inputFileStream = new FileStream(Path.GetFullPath("Data/Template.docx"), FileMode.Open, FileAccess.ReadWrite))
+{
+    //Open the template Word document.
+    using (WordDocument document = new WordDocument(inputFileStream, FormatType.Docx))
+    {
+        //Retrieve the first section of the document.
+        IWSection section = document.LastSection;
+        //Get the first table in the section.
+        WTable table = section.Body.Tables[0] as WTable;
+        //Access the specific cells by their indices.
+        WTableCell cell1 = table[1, 1];
+        WTableCell cell2 = table[2, 2];
+        //Clear the contents of the first cell.
```

Comment:
//Clear the contents of the cell (second row, second cell).

Change similar in all places of this sample

Reply:
Addressed